### PR TITLE
Added slovene translation

### PR DESCRIPTION
**Pull request by zuzexx**
## Description

Added a translation for the Slovene language.

-  changed `mobileapp/www/index.html` file
-  changed `web/locales-config.json` file
-  added `sl.json`  file with translations

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
I thought I'd just provide some insight why this has not been merged yet. We are currently rewriting the frontend and to avoid further merge conflicts and make everything a little simpler I will merge this after the new frontend has been released, we are targeting a release for the end of the month right now.

**Comment by zuzexx** [comment]
Hey, 
no problem at all. :)

**Comment by VIKTORVAV99** [comment]
I'm going to try and get this merged later today. I'll solve the merge conflicts and see if there is anything else we need to update since we merged the big re-write.

File: web/index.html
Comment (VIKTORVAV99) [inline web/index.html:66]: @madsnedergaard I assume we should and new languages here as well?

**Review by VIKTORVAV99 — APPROVED** [review]
I'll have to trust you on the translations but things look good to me!

PS: I cleaned up the PR after the re-write merge so it's all good to go, just want to double check something with Mads first.

File: web/index.html
Comment (madsnedergaard) [reply web/index.html:66]: Yup! And thanks for checking :)

Comment (madsnedergaard) [reply web/index.html:66]: And in one more place, I'll just do it

**Review by madsnedergaard — APPROVED** [review]
I used https://www.internationalphoneticalphabet.org/languages/language-names-in-native-language/ to find language in native language - so hope it's correct :)

**Pull request merged**
